### DISCUSSION TITLE
Using LLama3.1 with Function calling

### DISCUSSION THREAD

**Discussion by taha-ghadirian** [microsoft/semantic-kernel#7500] [Q&A] [answered]
Now LLama3.1 supports function calling and tools, along with  other tools like Ollama or Together.Ai, can we use it in semantic kernel for auto plugin calls? 
Is there any plan to add LLama to supported connectors?

**Comment by rogerbarreto** [comment, marked as answer]
Thanks for opening this issue!

Firstly we are targeting to release the Ollama Connector and get it merged into `main` soon and after that improve it including the task below which addresses also this discussion point! 😊

- #7442

**Reply by PaulaScholz** [reply]
LMStudio is also a preferred executive for making an OpenAI compatible server.  Do look into this, won't you?
